src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -766,426 +766,17 @@ public async Task GetGeneralInfoWithExceptionReturnsBadRequest()
         }
 
         [Fact]
-        public async Task GetHistoryWithoutAddressesReturnsEmptyModel()
```

Comment:
All these tests were moving to one encompassing integration test as we can't mock results any longer due to us query the DB directly with SQL.